src/components/WorkshopPage/Workshop.js

```diff
@@ -3,42 +3,57 @@ import ReactPlayer from 'react-player/youtube';
 import Button from '@material-ui/core/Button';
 import Grid from '@material-ui/core/Grid';
 import Typography from '@material-ui/core/Typography';
+import Box from '@material-ui/core/Box';
 import PropTypes from 'prop-types';
 
-function Workshop(props) {
+function Workshop({ title, youtube, author, description, readme, slides }) {
 	return (
 		<Grid item xs={12} sm={8} md={6}>
 			<ReactPlayer
 				style={{ borderRadius: '10px', overflow: 'hidden' }}
-				controls={true} width='100%' url={props.url} />
+				controls={true} width='100%' url={youtube} />
 			<hgroup style={{ marginTop: '1em' }}>
 				<Typography variant='h5' component='h3'>
-					{props.title}
+					{title}
 				</Typography>
 				<Typography variant='subtitle1' component='h4' style={{
 					textTransform: 'uppercase',
 					fontSize: '1em',
 					letterSpacing: '.5px'
 				}}>
-					{'Taught by: ' + props.author}
+					{'Taught by: ' + author}
 				</Typography>
 			</hgroup>
 			<Typography variant='body1' style={{ paddingTop: '0.5em', paddingBottom: '0.5em' }}>
-				{props.description}
+				{description}
 			</Typography>
-			<Button variant='contained' disableElevation color="secondary" component='a'
-				href={props.url} target='_blank' rel='noreferrer noopener'
-				style={{ textTransform: 'none', padding: '4px 1.5em', maxWidth: 'fit-content' }}>
-				See Video
-			</Button>
+			<Box component="span" display="flex" justifyContent="space-between">
```

Comment:
`space-between` looks a little, weird. Could you try aligning all of them to the left or right?

Reply:
Not sure if it's the alignment that looks odd if it just 3 `contained` buttons side-by-side that look odd.

Perhaps another button variant will make it look better? Let's experiment with the [`text` button variant](https://mui.com/components/buttons/#text-button) and see how that looks.

Reply:
the text button variant looks terrible if its aligned left, so I center aligned it with some padding in between. compare this to the contained button aligned left
<img width="942" alt="Screen Shot 2022-01-26 at 10 07 12 PM" src="https://user-images.githubusercontent.com/41168140/151302493-c1158487
<img width="938" alt="Screen Shot 2022-01-26 at 10 07 43 PM" src="https://user-images.githubusercontent.com/41168140/151302686-522509e6-4a6c-4d26-b34a-e8742357f37b.png">
-815f-4d3a-9afd-73533c11b3c0.png">

Reply:
<img width="942" alt="Screen Shot 2022-01-26 at 10 07 12 PM" src="https://user-images.githubusercontent.com/41168140/151302842-2e9e1e30-f27b-44f8-9618-10d04af7486b.png">

Reply:
Could you try using material ui icons?
https://v4.mui.com/components/icons/#icons these are the docs

These are the possible icons:
https://mui.com/components/material-icons/
We can do the youtube icon for the video, the github icon for the readme, and some other icon for slides (I don't think there is a perfect one for us, but try to look around and see what fits best) ?

Reply:
Also for the center align, you probably want to look at `space-around` and `space-evenly` for the `justify-content`

Reply:
there's very little visible difference between space-around and space-evenly, so I'm using space around. This is how the text buttons with material ui icons look. I'm debating between icons for the slides.
![Screen Shot 2022-01-27 at 11 33 07 AM](https://user-images.githubusercontent.com/41168140/151430716-bf61fb25-dd3d-4f23-8ed1-d57b612df604.png)
![Screen Shot 2022-01-27 at 11 34 58 AM](https://user-images.githubusercontent.com/41168140/151431411-a0869584-44f9-4149-9920-6fb2e991b214.png)